src/MLJMultivariateStatsInterface.jl

```diff
@@ -71,24 +71,24 @@ const SubspaceLDA_DESCR = """
     projection matrix `P = W*L` that projects a feature matrix `Xtrain` onto a lower
     dimensional space of dimension `nc - 1` such that the trace of the transformed
     between-class scatter matrix(`Pᵀ*Sb*P`) is maximized relative to the trace of the
-    transformed within-class scatter matrix (`Pᵀ*Sw*P`). The projection matrix is scaled 
-    such that `Pᵀ*Sw*P = mult*I` or `Pᵀ*Σw*P=mult/(n-nc)*I` (where `n` is the number of 
-    training samples, mult` is  one of `n` or `1` depending on whether `Sb` is normalized, 
-    `Σw` is the within-class covariance matrix, and `nc` is the number of unique classes 
+    transformed within-class scatter matrix (`Pᵀ*Sw*P`). The projection matrix is scaled
+    such that `Pᵀ*Sw*P = mult*I` or `Pᵀ*Σw*P=mult/(n-nc)*I` (where `n` is the number of
+    training samples, mult` is  one of `n` or `1` depending on whether `Sb` is normalized,
+    `Σw` is the within-class covariance matrix, and `nc` is the number of unique classes
     in `y`) and also obeys `Wᵀ*Sb*p = λ*Wᵀ*Sw*p`, for every column `p` in `P`.
-    Predicted class posterior probability for feature matrix `Xtest` are derived by 
-    applying a softmax transformation to a matrix `Pr`, such that  rowᵢ of `Pr` contains 
-    computed distances(based on a distance metric) in the transformed space of rowᵢ in 
+    Predicted class posterior probability for feature matrix `Xtest` are derived by
+    applying a softmax transformation to a matrix `Pr`, such that  rowᵢ of `Pr` contains
+    computed distances(based on a distance metric) in the transformed space of rowᵢ in
     `Xtest` to the centroid of each class.
     """
 const BayesianSubspaceLDA_DESCR = """
-       Bayesian Multiclass linear discriminant analysis. Suitable for high dimensional data 
-    (Avoids computing scatter matrices `Sw` ,`Sb`). The algorithm learns a projection 
-    matrix `P = W*L` (`Sw`), that projects a feature matrix `Xtrain` onto a lower 
-    dimensional space of dimension `nc-1` such that the trace of the transformed 
-    between-class scatter matrix(`Pᵀ*Sb*P`) is maximized relative to the trace of the 
-    transformed within-class scatter matrix (`Pᵀ*Sw*P`). The projection matrix is scaled 
-    such that `Pᵀ*Sw*P = mult*I` or `Pᵀ*Σw*P=mult/(n-nc)*I` (where `n` is the number of 
+       Bayesian Multiclass linear discriminant analysis. Suitable for high dimensional data
+    (Avoids computing scatter matrices `Sw` ,`Sb`). The algorithm learns a projection
+    matrix `P = W*L` (`Sw`), that projects a feature matrix `Xtrain` onto a lower
+    dimensional space of dimension `nc-1` such that the trace of the transformed
+    between-class scatter matrix(`Pᵀ*Sb*P`) is maximized relative to the trace of the
+    transformed within-class scatter matrix (`Pᵀ*Sw*P`). The projection matrix is scaled
+    such that `Pᵀ*Sw*P = mult*I` or `Pᵀ*Σw*P=mult/(n-nc)*I` (where `n` is the number of
```

Comment:
i think we can just delete these constants, right? We are making them redundant.